guides/image-sources/prov-installation-media.iuml

```diff
@@ -0,0 +1,6 @@
+loop
+   Host -> Foreman : requests installation media
+   Foreman -> Proxy : requests installation media
```

Comment:
There are three variants:
1. Foreman with Katello, provisioning RHEL machine with kickstart repo (downloading inst media from proxy)
2. Foreman with Katello, provisioning other OS (downloading installation media from the cloud)
3. Foreman without Katello (downloading installation media from the cloud)

Here, we are describing 1st case; should we also cover the other two?

Reply:
I tried hiding all three in there, but I wrongly thought that 2. and 3. also involve SmartProxy. How about keeping just the following and not worry about where Foreman gets the installation media from?
```
loop
   Host -> Foreman : requests installation media
   Foreman -> Host : provides installation media
end
```

Reply:
> just the following and not worry about where Foreman gets the installation media

I agree; just one note: Foreman asks Smart Proxy to download it, and Smart Proxy will do the heavy work.

Reply:
> > just the following and not worry about where Foreman gets the installation media
> 
> I agree; just one note: Foreman asks Smart Proxy to download it, and Smart Proxy will do the heavy work.

That means that it's already correct as it is, doesn't it?

Reply:
Oh shoot I missed that somehow omg. Yes, requests vs provides is good.